packages/desmos-std/src/profiles/query.rs

```diff
@@ -13,34 +13,18 @@ pub enum ProfilesQuery {
         receiver: Addr,
         pagination: Option<PageRequest>,
     },
-    Relationships {
-        user: Addr,
-        subspace_id: Uint64,
-        pagination: Option<PageRequest>,
-    },
-    Blocks {
-        user: Addr,
-        subspace_id: Uint64,
-        pagination: Option<PageRequest>,
-    },
     ChainLinks {
-        user: Addr,
+        user: Option<Addr>,
```

Comment:
Any reason to change to Option<T> here?
In the codec, I remember that it shows empty string as null.

Reply:
After the desmos core refactor, both `QueryUserChainLinks` and `QueryUserAppLinks` have been merged into `QueryChainLinks` and `QueryAppLinks` queries, making their parameters all optional inside them.

Ref: https://github.com/desmos-labs/desmos/blob/master/proto/desmos/profiles/v2/query_chain_links.proto

So since these changes applied to the core. I think it's better from a consistency and logic perspective to also have them here as well.   
Doing this we will allow devs to pass `None` to each parameter if for instance they want to retrieve the whole `Chain` or `App` Links, instead of forcing them to pass empty strings.

Reply:
As I know the `None` will be serialized to `null` in json so that the `ChainLinks` query json is like:
```json
{
    "user": null,
    "target": null,
    "chain_name": null
}
```

Is it available to be unmarshaled to the `QueryChainLinkRequest`?

Reply:
@bragaz Ok, I just checked it can work properly. It is good to me with the changes.